### PR TITLE
MAINT: Dict litteral

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: scipy/weave/c_spec.py [hunk 1]
```diff
@@ -301,36 +300,30 @@ def c_to_py_code(self):
...
+                  'b': 'char', 'B': 'npy_ubyte',
+                  'h': 'short', 'H': 'npy_ushort',
+                  'i': 'int', 'I': 'npy_uint',
+                  '?': 'bool',
+                  'l': 'long', 'L': 'npy_ulong',
+                  'q': 'npy_longlong', 'Q': 'npy_ulonglong'}
+
+
 # !! hmmm. The following is likely unsafe...
-num_to_c_types[long] = 'npy_longlong'
```
Comment (rgommers) [inline scipy/weave/c_spec.py]: Here you lost the link between this line and the comment above. Can you move the comment up and explain that that's about ``long --> npy_longlong``?

File: scipy/signal/wavelets.py [hunk 2]
```diff
@@ -167,8 +167,8 @@ def cascade(hk, J=7):
     if sm < 0:  # need scaling function to integrate to 1
         v = -v
         sm = -sm
-    bitdic = {}
-    bitdic['0'] = v / sm
+    bitdic = {'0': v / sm,
+              }
```
Comment (rgommers) [inline scipy/signal/wavelets.py]: put this on a single line?

**Comment by rgommers** [comment]
looks good apart from 2 minor things to fix up

**Comment by sieben** [comment]
@rgommers done :)

**Comment by rgommers** [comment]
@sieben now the PEP8 checker is unhappy so the TravisCI check failed:

    scipy/weave/c_spec.py:306:40: E261 at least two spaces before inline comment
    1       E261 at least two spaces before inline comment

can you fix that and ping me again?

**Comment by sieben** [comment]
Done :)

**Comment by rgommers** [comment]
Thanks @sieben, merged.

**Pull request merged**